src/gba/GBA.cpp

```diff
@@ -1712,11 +1712,21 @@ const char* GetSaveDotCodeFile()
     return saveDotCodeFile;
 }
 
+void ResetLoadDotCodeFile()
+{
+    loadDotCodeFile = strdup("");
```

Comment:
@s520 Is this leaking memory? I believe you should `if (loadDotCodeFile) free(loadDotCodeFile)` before the `strdup`.

Reply:
Oops, that's right.
I will fix it.

Reply:
Please check that I have fixed it.